src/test/java/org/openhab/tools/analysis/checkstyle/test/PomXmlCheckTest.java

```diff
@@ -98,18 +99,31 @@ public void testMasterPom() throws Exception {
         verify(createChecker(config), testFileAbsolutePath, ArrayUtils.EMPTY_STRING_ARRAY);
     }
 
+    @Test
+    public void testDifferentVersion() throws Exception {
+        verifyFile("different_version_directory", false, 0,
+                "The manifest version did not match the requirements " + MANIFEST_REGEX);
+    }
+
     @Override
     protected DefaultConfiguration createCheckerConfig(Configuration config) {
         DefaultConfiguration defaultConfiguration = new DefaultConfiguration("root");
         defaultConfiguration.addChild(config);
         return defaultConfiguration;
     }
 
-    private void verifyPomXmlFile(String testDirectoryName, int expectedLine, String expectedMessage) throws Exception {
+    private void verifyFile(String testDirectoryName, boolean isPomXmlFile, int expectedLine, String expectedMessage)
+            throws Exception {
         String testDirectoryAbsolutePath = getPath(POM_XML_CHECK_TEST_DIRECTORY_NAME + "/" + testDirectoryName);
         File testDirectory = new File(testDirectoryAbsolutePath);
         File[] testFiles = listFilesForDirectory(testDirectory, new ArrayList<File>());
-        String testFilePath = testDirectory.getPath() + File.separator + POM_XML_FILE_NAME;
+        String testFilePath = null;
+        if (isPomXmlFile) {
+            testFilePath = testDirectory.getPath() + File.separator + POM_XML_FILE_NAME;
+        } else {
+            testFilePath = testDirectory.getPath() + File.separator + "META-INF" + File.separator + "MANIFEST.MF";

```

Comment:
You can use the constants placed in `org.openhab.tools.analysis.checkstyle.api.CheckConstants`.